File: 62.py
k = input()
b = int(k[1])
a = ord(k[0])
 
summa = 0
 
if 66 <= a + b and 80 >= a + b:
    summa += a + b
    if summa % 2:
        print("WHITE")
    elif summa % 2 == 0:
        print("BLACK")

File: 62.c
#include<stdio.h>

int main(){

    char a;
    int  b, collect=0;
    scanf("%c%d",&a,&b);

    if(66<=a+b && 80>=a+b){
        collect=a+b;
        if(collect%2==1){
                printf("WHITE");
        }
        else if(collect%2==0){
                printf("BLACK");
        }
    }
    
    return 0;
}
